sale_order_delivery_date/views/stock_picking_views.xml

```diff
@@ -8,7 +8,14 @@
         <field name="arch" type="xml">
             <xpath expr="//field[@name='scheduled_date']" position="after">
                 <field name="delivery_due_date" attrs="{'invisible':[('picking_type_code','!=','outgoing')]}"/>
-                <field name="date_delivered" attrs="{'required':[('picking_type_code','=','outgoing'),('state','=','done')]}" readonly="0"/>
+                <field
+                    name="date_delivered"
+                    attrs="{
+                        'required':[('picking_type_code','=','outgoing'),('state','=','done')],
+                        'invisible':[('picking_type_code','!=','outgoing')]
+                        }"
+                    readonly="0"
+                />
```

Comment:
Can we apply the similar design as what we did here to these fields: https://github.com/qrtl/rpl-private/blob/70b16dfff995b8532d4f57a9525924595e5be797/stock_picking_serial_verify/views/stock_picking_views.xml#L10-L11

See how `need_serials` is computed here: https://github.com/qrtl/rpl-private/blob/70b16dfff995b8532d4f57a9525924595e5be797/stock_picking_serial_verify/models/stock_picking.py#L101-L109

We can name the field `is_customer_outgoing` or something instead of `need_serials`.

`delivery_due_date`and `date_delivered` should only be visible/required when `is_customer_outgoing` is true.

Reply:
Thank you.
I corresponded it, and worked as expected.